src/NorthwindWebsite/API/Controllers/RootController.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace NorthwindWebsite.API.Controllers;
+
+[Route("/api")]
```

Comment:
I ain't sure `/` is required here since your other controllers have addition postfix